app/components/cases/subcases/mandatee-selector-panel.js

```diff
@@ -119,6 +119,18 @@ export default class MandateeSelectorPanel extends Component {
         );
         if (mandatee) {
           selectedMandatees.push(mandatee);
+        } else {
+          const oldMandateePerson = oldMandatee.person;
```

Comment:
This needs to be `yield`ed. This code needs to move out of a `forEach` and into a normal `for`-loop.